### ISSUE TITLE
your website doesn't work on IE11

### ISSUE THREAD

**Issue by giufurlan** [FortAwesome/Font-Awesome#16628]
I'm trying to open https://fontawesome.com/ on IE 11 but I can see only a white page.
My IE version is 11.719.18362.0

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

I don't know if we support IE11 on the documentation website

@robmadole ?

**Comment by robmadole** [comment]
Yeah this is a known issue. Our support for IE10 and IE11 has been really dodgy from the beginning because we just don't routinely test it.

@giufurlan what restrictions do you have that you are using IE11? We're considering dropping support for IE for the site (not Font Awesome itself)

**Comment by giufurlan** [comment]
The web application where I'm working need to support ie11, I upgraded fontawesome from 4.7 to latest, I was looking to understand if everything working, and I saw you website didn't work.
I was hoping to do what I liked by reporting it to you
